server/plugin/configuration.go

```diff
@@ -27,6 +27,7 @@ type Configuration struct {
 	EnterpriseBaseURL       string
 	EnterpriseUploadURL     string
 	EnableCodePreview       string
+	EnableLeftSidebar       bool
```

Comment:
Please order this in the same way as in `plugin.json`

Reply:
done.